replicate/common/namespaces.go

```diff
@@ -51,6 +62,9 @@ func (nw *NamespaceWatcher) create(client kubernetes.Interface, resyncPeriod tim
 			resyncPeriod,
 			cache.ResourceEventHandlerFuncs{
 				AddFunc: namespaceAdded,
+				UpdateFunc: func(old, new interface{}) {
+					namespaceUpdated(old, new)
+				},
```

Comment:
Do we actually need the intermediate anonymous function, or can we just pass `namespaceUpdated` directly?

```suggestion
				UpdateFunc: namespaceUpdated(old, new),
```